tests/integration/test_repository.py

```diff
@@ -1033,6 +1033,9 @@ async def test_repo_filter_collection(
     existing_obj = await maybe_async(author_repo.list(CollectionFilter(field_name="id", values=[second_author_id])))
     assert existing_obj[0].name == "Leo Tolstoy"
 
+    no_obj = await maybe_async(author_repo.list(CollectionFilter(field_name="id", values=[])))
```

Comment:
I think it would be better to have this as its own test function

Reply:
Cool.  I'll update this morning before getting out a patch release.